src/App.tsx

```diff
@@ -1,24 +1,183 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { addTodos, deleteTodos, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorTypes } from './types/ErrorTypes';
+import { ErrorMessage } from './components/ErrorMessage';
+import { TodoItem } from './components/TodoItem';
 
-const USER_ID = 0;
+const USER_ID = 10905;
 
 export const App: React.FC = () => {
+  const [title, setTitle] = useState<string>('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedFilter, setSelectedFilter] = useState('All');
+  const [loadingTodosId, setLoadingTodosId] = useState<number[]>([0]);
+  const [isDisabled, setIsDisabled] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [errorMessage, setErrorMessage] = useState<ErrorTypes>();
+
+  const titleField = useRef<HTMLInputElement>(null);
+
+  const handleChangeTitle = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(e.target.value);
+  };
+
+  const showAndDeleteError = () => {
+    setIsError(true);
+    setTimeout(() => setIsError(false), 3000);
```

Comment:
It's a good practice to use clean-up functions on the component unmount
https://bobbyhadz.com/blog/react-cleartimeout

Reply:
Done, but please correct if I have done it uncorrectly or there is a simplier way and more productive way.